e2e/cypress/integration/api/iam/06_projects_api_compliance_update.ts

```diff
@@ -0,0 +1,268 @@
+import { describeIfIAMV2p1 } from '../../constants';
```

Comment:
This is the main change for this PR. Added the compliance tests for project update tagging. This tests the painless scripts.